public/scss/_base.scss

```diff
@@ -128,6 +128,31 @@ li:not(:last-of-type) {
   }
 }
 
+.confirmation-table {
+  width: 100%;
+  margin-bottom: $space-sm;
+  color: white;
+  background-color: $color-blue-light;
+  border: 3px solid $color-solid-blue;
```

Comment:
I think the border looks odd (looks like a focus style). 

But also I don't want to leave code here all day, so let's catch up with @brianhendrick1 when he's back.